CONTRIBUTING.md

```diff
@@ -30,3 +36,133 @@ to review and merge.
 For UI changes, please include screenshots to help reviewers.
 
 Once approved, the author may merge the Pull Request.
+
+
+## "Sprint" rituals
```

Comment:
I don’t think we want to talk about sprints.  I think we can rephrase this without talking about sprints

Reply:
Sure, do you have a suggestion? Just "Rituals"?